pkg/address/constants.go

```diff
@@ -0,0 +1,65 @@
+package address
+
+import (
+	"encoding/base32"
+	"errors"
+
+	"github.com/minio/blake2b-simd"
+)
+
+func init() {
+
+	var err error
+
+	TestAddress, err = NewActorAddress([]byte("satoshi"))
+	if err != nil {
+		panic(err)
+	}
+
+	TestAddress2, err = NewActorAddress([]byte("nakamoto"))
+	if err != nil {
+		panic(err)
+	}
+}
+
+var (
+	// TestAddress is an account with some initial funds in it.
+	TestAddress Address
+	// TestAddress2 is an account with some initial funds in it.
+	TestAddress2 Address
```

Comment:
These will introduce coupling between tests, not just in one project but now across multiple repos! Please remove from this repo, the implementations can redefine values if needed.